### PR TITLE
Added Expect-CT to the list and allowed the use of colour profiles to not bias against people who don't use dark mode

### PR DESCRIPTION
**Pull request by tautology0**

### CODE REVIEW

File: shcheck.py [hunk 1]
```diff
@@ -242,8 +256,9 @@ def main(options, targets):
             targets = f.read().splitlines()
         
 
-
+    json_out=[]
```
Comment (meliot) [inline shcheck.py:259]: I think this should be a dict, otherwise the JSON output is not JSON but a list

**Comment by meliot** [comment]
Hi! Thank you for the contribution 😄 I left some comments to make the PR more clean, can you also please resolve the conflicts? I'll then merge it.

File: shcheck.py (hunk 1 above)
Comment (tautology0) [reply shcheck.py:259]: You're right, I'm not certain the best route, you could encapsulate them in one dictionary, with an array that is {"results":[ ... ]}

**Comment by tautology0** [comment]
BTW I originally removed these lines (and added them back in at the commit) as they didn't actually do anything when I ran it on my Fedora VM:

```
    # Disabling printing if json output is requested
    if json_output:
        global json_headers
        sys.stdout = open(os.devnull, 'w')
```

**Pull request merged**